frontend/express/public/javascripts/countly/countly.common.js

```diff
@@ -3351,4 +3351,24 @@
         return tmpAvgVal.toFixed(2);
     }
 
+    /**
+    * Async for each function
+    * @param {array} array - array to loop
+    * @param {function} fn - callback function
+    * @param {function} atEnd - optional function to will work at the end
+    */
+    countlyCommon.asyncForeach = function(array, fn, atEnd) {
```

Comment:
And same function in 2 places?  use async.js
Like this:
https://github.com/Countly/countly-server/blob/9ffae2ac5a9afea4707f586fc65d659cb4ce7d31/bin/upgrade/18.01/scripts/delete_drill_meta.js#L24